### ISSUE TITLE
add sample indexes to report

### ISSUE THREAD

**Issue by FranBonath** [NationalGenomicsInfrastructure/anglerfish#89]
The report currently only shows undetermined indexes. Would be nice to also see the (Illumina) indexes for each sample, because it makes troubleshooting easier.

**Comment by remiolsen** [comment]
I'm sorry, what do you mean? Do your want an Illumina samplesheet or read counts as input? Or maybe something like the `anglerfish_dataframe.csv` which is a current feature but might need some updating / streamlining. Take a look at an example one:

[anglerfish_dataframe.csv](https://github.com/user-attachments/files/16662358/anglerfish_dataframe.csv)

**Comment by kedhammar** [comment]
Yes @FranBonath I would say opening the .csv dataframe in a spreadsheet would be the best option for troubleshooting. If you like I can make sure the dataframe is dumped in LIMS.
